docs/learn/dapp-staking/dapp-staking-protocol.md

```diff
@@ -328,6 +328,16 @@ When a dApp has the minimum stake amount to just enter the tier, its rank will b
 If a dApp is halfway to the next tier, its rank is **5** (five). If they are almost at the next tier, their rank is **9** (nine).
 If a dApp reaches the threshold to enter the next tier but there is no empty slot in that tier and the dApp remains in the current tier, they will get the highest rank of **10** (ten).
 
+Formula to calculate rank threshold is as follows ${rank\_threshold} = {\frac{{upper\_tier\_treshold} - {tier\_threshold}}{10}}$
```

Comment:
Would more correct be to say that `rank_threshold` is the number you calculate in your formula + `tier_threshold`?

This is more of a `rank_offset` I guess.

Reply:
offset doesn't make sense to me. it's more like a step threshold

Reply:
I'd avoid mentioning `threshold` word in the Ranking chapter. It is already complex enough how all of it works. In the Oracle section you talk about thresholds in generic terms not mentioning is it Ranking or Tier threshold.
Better use some other word.

Reply:
I guess it's ok then.